src/sentry/web/frontend/auth_login.py

```diff
@@ -159,7 +159,9 @@ def handle(self, request, *args, **kwargs):
     def get(self, request, *args, **kwargs):
         next_uri = self.get_next_uri(request, *args, **kwargs)
         if request.user.is_authenticated():
-            return self.handle_authenticated(request, *args, **kwargs)
+            # we need to allow re-auth for superusers
+            if not request.user.is_superuser or request.is_superuser():
```

Comment:
this is confusing. Can we add a comment why this is necessary?

Reply:
i feel like the comment that i put there gives that, but i expanded it

but now i have to wait another 20m on tests -.-

Reply:
Nevermind. I read `request.is_superuser`.